addons/website/static/src/scss/website.scss

```
@@ -1591,6 +1591,9 @@ $ribbon-padding: 100px;
 
 // Cookies Bar
 #website_cookies_bar {
+    min-height: 0px;
+    height: 0px;
```

I didn't get why you needed the `min-height: 0;` until I saw what it solves. But that behavior is really weird, do you know why it works like that?
Anyway, I'd suggest to reduce that to one line which feels a bit less hackish:
```suggestion
    position: absolute;
```
(`position: fixed;` works too, basically the idea is to remove it from the flow of the page so it takes no space.)

---

> But that behavior is really weird, do you know why it works like that?

If you mean why does this element receive such style, see https://github.com/odoo/odoo/commit/cfd53b8fae3ad9f677df49a03fe6d2945dbb5da2
> Since the refactoring of the editor done at [1], the cookies bar popup
is receiving a `contenteditable=true` attribute, making it receive the
chrome default `height` style for such elements.
The problem is that if this cookies bar's modal is hidden, the bar will
be shown as a thin empty white bar.

Funny thing is, there is no way the bug fixed in the current PR was there at that point, it appeared again later and only when the cookies bar is not hidden.

Maybe this commit could be mentioned in this commit to tie everything together, as they fix the same issue (white bar because of browser style on contenteditable=true).

---

Ah no I wasn't clear, that was very much just a CSS question on my part.
I meant that `height: 0;` is not enough. If you don't put `min-height: 0;` with it, the container gets a height when you click inside the popup. Which bugs me, because 1) how would _min-height_ be lesser than 0 when the _height_ is 0, and 2) why does setting a _min-height_ prevent the container from having any height at that moment, while _height_ doesn't (and _max-height_ neither).

---

>  I didn't get why you needed the min-height: 0; until I saw what it solves. But that behavior is really weird, do you know why it works like that?
> Anyway, I'd suggest to reduce that to one line which feels a bit less hackish:

@robinlej Thanks for the check :+1: But I think the `position: absolute` is more hackish than the current diff ? Simply because there is no direct reason to change the position while there is a good direct reason to set `height` and `min-height` to `0`: you can see in [this code](https://github.com/odoo/odoo/blob/15.0/addons/website/static/src/scss/website.edit_mode.scss#L55-L57) that the `.o_editable` elements has also a minimum height when it is focused (which causes the weird behavior you noticed when only removing the `min-height: 0;`).
Remark: you can find in this [commit](https://github.com/odoo/odoo/commit/6007264afb83dcfed1693d72179cb156ff2254b6#diff-f338780f594d6b1f46eda5d3fb9fed86f3c3c8f65522fb93e639534e9d92cdb2R212) a similar issue with the `#o_shared_blocks` container fixed with the same logic :+1:

@rdeodoo Thanks for mentioning the commit, I will areference it in the commit message :+1:

---

> you can see in [this code](https://github.com/odoo/odoo/blob/15.0/addons/website/static/src/scss/website.edit_mode.scss#L55-L57) that the .o_editable element has also a minimum height when it is focused

Ah, so that's why. Thanks.

> I think the position: absolute is more hackish than the current diff

Well I don't have strong feelings so it's good for me. :) I'll r+ once you've updated the message then.